readme.md

```diff
@@ -86,9 +86,22 @@ const honey = useBearStore((state) => state.honey)
 
 If you want to construct a single object with multiple state-picks inside, similar to redux's mapStateToProps, you can tell zustand that you want the object to be diffed shallowly by passing the `shallow` equality function.
 
+To use a custom equality function, you need `createWithEqualityFn` from `zustand/traditional`. It takes the second argument for the default equality function.
```

Comment:
I feel it's a bit confusing — "to use a custom equality function" and then "for the default equality function."

Is the default eq fn a default for that store, but you can still override it later? If so, I suggest:

```suggestion
To use a custom equality function that will become a new default for the store, you need `createWithEqualityFn` from `zustand/traditional`.
Pass an equality function as a second argument.
```

Reply:
Thanks for the suggestion. Let me clarify and maybe you have some other suggestions.
- `create` (in v5) doesn't allow changing equality function. (It's always `Object.is` equivalent.)
- To use custom equality function, `createWithEqualityFn` is necessary.
  - The signature is `const useMyStore = createEqualityFn(..., defaultEqualityFn)`.
  - `useMyStore(selector, equalityFn)` can override the default equality function.
  - If you specify `Object.is` for the second argument of createEqualityFn, it behave the same as `create` < v4.4.0.

Reply:
Alright, so my proposal:

```suggestion
To use a custom equality function that will become a new default for the store, you need `createWithEqualityFn` from `zustand/traditional`.
Pass a desired equality function as a second argument, like so:
```

Reply:
What I was concerning is the backward compatibility with previous version. But, I think it's not super important.

Reply:
> for the store

It's not for the store (which is still Objet.is), but it's for the hook.

Reply:
"To use a custom equality function, you need `createWithEqualityFn` instead of `create`. Usually, specify `Object.is` at the second argument for the default equality function, but it's configurable." How about that?

Reply:
"To use a custom equality function, you need `createWithEqualityFn` instead of `create`. Usually you want to specify `Object.is` as the second argument for the default equality function, but it's configurable."

Reply:
Thanks. I'll update it later.